demo/discourse.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+import logging
+
```

Comment:
import requests at the module level since you use it in multiple functions, you can just `import requests` so that we know what `requests.get` refers to instead of `r.get`